File: playground/BOJ/boj_1546_average/boj_1546_average.py
'''----------------------------------------------------
Sub  : [BOJ] 평균
Link : https://www.acmicpc.net/problem/1546
Level: 브론즈 1
Tag  : Python,
Memo
----------------------------------------------------'''

import sys
input = sys.stdin.readline

# 입력
n = int(input().rstrip())
scores = list(map(int, input().rstrip().split()))

#최대값
max_score = max(scores)

average_score = sum((score / max_score) * 100 for score in scores) / n

# 출력
print(average_score)

File: Online_Judge_Problems/BOJ/boj_1546_average/boj_1546_average.js
/*-----------------------------------------------------
Sub  : [BOJ] 평균
Link : https://www.acmicpc.net/problem/1546
Level: 브론즈 1
Tag  : JS,
Memo
-----------------------------------------------------*/

const input = require('fs').readFileSync(0, 'utf-8').trim().split('\n');

// 입력
const n = parseInt(input[0], 10);
const scores = input[1].split(' ').map(Number);

// 최댓값
const maxScore = Math.max(...scores);

// 고친 점수들의 합
const sum_score = scores.reduce((acc, score) => acc + (score / maxScore) * 100, 0);

// 평균
const average_score = sum_score /n

// 출력
console.log(average_score);
